N/NOMAD/build_tarballs.jl

```diff
@@ -37,6 +39,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = Dependency[]
```

Comment:
```suggestion
dependencies = [
```
:wink:

Reply:
Oups, solved :man_technologist:

Reply:
You can also just click on the "Commit suggestion" button instead of having to do everything manually :slightly_smiling_face: